src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -30,7 +34,13 @@
   protected String sentinelPassword;
   protected String sentinelClientName;
 
-  protected final Set<MasterListener> masterListeners = new HashSet<>();
+  protected boolean isMasterSslEnabled;
+  protected boolean isSentinelSslEnabled;
+  protected SSLSocketFactory sslSocketFactory;
+  protected SSLParameters sslParameters;
+  protected HostnameVerifier hostnameVerifier;
+
+  protected Set<MasterListener> masterListeners = new HashSet<MasterListener>();
```

Comment:
Keep final keyword from earlier.
```suggestion
  protected final Set<MasterListener> masterListeners = new HashSet<>();
```

Reply:
reverted the change